ctp2_code/ai/diplomacy/dstateevent.cpp

```diff
@@ -34,12 +34,12 @@ STDEHANDLER(InitDStateEvent)
 		return GEV_HD_Continue;
 
 	Diplomat & diplomat = Diplomat::GetDiplomat(playerId);
-	AiState state;
+	AiState thisstate;
```

Comment:
Why is state changed to 'thisstate'? Is it needed from a technical point of view? Personally I do not like names prefixed with 'the' or 'this' or something else that is not functionally describing the variable.

This also makes the changelist much bigger than maybe needed.

Reply:
Agreed - this is an uneccesary change and was motivated by misreading a different ld error on Fedora 32 when configured with --enable-debug due to int state being defined in both ldl.l and slic.l

I will revert this change.